test/cleanup.test.js

```diff
@@ -1,187 +1,284 @@
 'use strict';
 
 /* eslint-disable no-console */
-process.argv[4] = '.';
+
+const child_process = require('child_process');
 const AWS = require('@mapbox/mock-aws-sdk-js');
 const sinon = require('sinon');
 const test = require('tape');
-const file = require(`${__dirname}/../scripts/cleanup`);
+const cleanup = require(`${__dirname}/../scripts/cleanup`);
+
 const region = 'us-east-1';
 const repo = 'some-repo';
 const token = 'sometoken';
-const error = 'some error';
-const success = 'some success';
+const tmpdir = '/path/to/cloned/repo';
+
 const imagesNoToken = require(`${__dirname}/fixtures/imagesNoToken.test.json`);
 const imagesToken = require(`${__dirname}/fixtures/imagesToken.test.json`);
 const imageDetails = imagesNoToken.imageDetails.concat(imagesToken.imageDetails);
-const imageIds = [{ imageDigest: imagesNoToken.imageDetails[1].imageDigest }];
-
-test('mockSpawn', (t) => {
```

Comment:
I really don't like this pattern where the first test in a series sets up mocks used by latter tests. Its very important that each individual test can run in complete isolation from any other one.